Metagenomics/tutorials/mothur-miseq-sop.md

```diff
@@ -0,0 +1,1516 @@
+
+> ### :pencil2: Hands-on: Summary.single
+>
+> - **Tool:** Summary.single
+> - **Parameters:**
+>   - Set **share** to shared file from Make.shared
+>   - Set **calc** to `nseqs,coverage,sobs,invsimpson`
+>   - Set **size** to 2440
+{: .hands_on}
+
+The data will be outputted to a table called the *summary file*:
+
+```
+label   group   sobs          coverage    invsimpson   invsimpson_lci   invsimpson_hci  nseqs
+0.03    F3D0    167.000000    0.994697    25.686387    24.648040        26.816067       6223.000000
+0.03    F3D1    145.000000    0.994030    34.598470    33.062155        36.284520       4690.000000
+0.03    F3D141  154.000000    0.991060    19.571632    18.839994        20.362390       4698.000000
+0.03    F3D142  141.000000    0.978367    17.029921    16.196090        17.954269       2450.000000
+0.03    F3D143  135.000000    0.980738    18.643635    17.593785        19.826728       2440.000000
+0.03    F3D144  161.000000    0.980841    15.296728    14.669208        15.980336       3497.000000
+0.03    F3D145  169.000000    0.991222    14.927279    14.494740        15.386427       5582.000000
+0.03    F3D146  161.000000    0.989167    22.266620    21.201364        23.444586       3877.000000
+0.03    F3D147  210.000000    0.995645    15.894802    15.535594        16.271013       12628.000000
+0.03    F3D148  176.000000    0.995725    17.788205    17.303206        18.301177       9590.000000
+0.03    F3D149  194.000000    0.994957    21.841083    21.280343        22.432174       10114.000000
+0.03    F3D150  164.000000    0.989446    23.553161    22.462533        24.755101       4169.000000
+0.03    F3D2    179.000000    0.998162    15.186238    14.703161        15.702137       15774.000000
+0.03    F3D3    127.000000    0.994167    14.730640    14.180453        15.325243       5315.000000
+0.03    F3D5    138.000000    0.990523    29.415378    28.004777        30.975621       3482.000000
+0.03    F3D6    155.000000    0.995339    17.732145    17.056822        18.463148       6437.000000
+0.03    F3D7    126.000000    0.991916    13.343631    12.831289        13.898588       4082.000000
+0.03    F3D8    158.000000    0.992536    23.063894    21.843396        24.428855       4287.000000
+0.03    F3D9    162.000000    0.994803    24.120541    23.105499        25.228865       5773.000000
+```
+
+Interestingly, the sample coverages were all above 97%, indicating that we did a pretty good job of sampling the communities. Plotting the richness or diversity of the samples would show that there was little difference between the different animals or between the early and late time points. You could follow this up with a repeated-measures ANOVA and find that there was no significant difference based on sex or early vs. late.
+
+## Beta diversity
+
+> ### :pencil2: Hands-on: Beta diversity
+>
+> Let's calculate the similarity of the membership and structure found in the various samples. We'll do this
```

Comment:
Which beta diversity measure is used? Same question for alpha diversity? Is it possible to change the used measure?